src/components/Producten.jsx

```diff
@@ -0,0 +1,27 @@
+import ButtonPlusMin from "./ButtonPlusMin";
+export default function Producten ({
+        productName,
+        icon,
+        stateSetter,
+        stateValue}) {
+    function ofNull (stateValue){
+        if (stateValue === 0) {
+            return true;
+        }
+    }
+    return(
+        <article>
+            <p>{icon} {productName}</p>
+            <ButtonPlusMin
+                buttonText = " - "
+                buttonDisabled = {ofNull(stateValue)}
+                buttonActie = {() => stateSetter(stateValue - 1)}>
```

Comment:
Als je de huidige state wil gebruiken, dan kun je beter die gebruiken die meegegeven wordt
```suggestion
                buttonActie = {() => stateSetter(curr => curr - 1)}>
```
Hierdoor wordt er niet telkens een nieuwe functie aangemaakt in memory, omdat een waarde buiten de functie veranderd wordt